src/server/game/Warden/WardenWin.cpp

```diff
@@ -337,6 +338,15 @@ void WardenWin::HandleData(ByteBuffer &buff)
     {
         buff.rpos(buff.wpos());
         TC_LOG_WARN("warden", "%s failed checksum. Action: %s", _session->GetPlayerInfo().c_str(), Penalty().c_str());
+        if ((sIRC->BOTMASK & 2048) != 0 && sIRC->anchn.size() > 0)
```

Comment:
there are quite a few magic numbers used in the BOTMASK field, please use an enum